tools/ka-customizer

```diff
@@ -190,7 +190,7 @@ add_label('Choose a stock theme (overrides application theme):');
 let stock_theme_chooser = new Gtk.ComboBoxText({
     halign: Gtk.Align.START,
 });
-['', 'default', 'encyclopedia', 'news', 'buffet', 'preset_a', 'preset_b'].forEach((item) => {
+['', 'default', 'buffet', 'course', 'encyclopedia', 'escola', 'library_list', 'news', 'picard', 'preset_a', 'preset_b'].forEach((item) => {
```

Comment:
can we exclude picard theme here? that theme is not really meant for apps, and could cause confusion?